crates/bevy_ecs/src/storage/mod.rs

```diff
@@ -1,4 +1,24 @@
 //! Storage layouts for ECS data.
+//!
+//! This module contains the low level backing data stores for ECS. These all offer minimal and often
+//! unsafe APIs, exposed primarily for debugging and monitoring purposes.
```

Comment:
I like that you mentioned it was for "debugging and monitoring purposes". You clarified a question I had before I eve n thought to ask it.